### ISSUE TITLE
Chain Joining - How-to Doc

### ISSUE THREAD

**Issue by brentbenton** [interbit/interbit#283]

**Comment by brentbenton** [comment]
@btlewan is this done?

**Comment by btlewan** [comment]
@brentbenton No. I recently added conceptual information about chain joins, but this issue is specifically about "How To". I need to work up a tutorial that shows how to establish read joins and write joins.
